### DISCUSSION TITLE
Is it possible to restore a previous codespace?

### DISCUSSION THREAD

**Discussion by LorrinWWW** [community/community#56555] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I have some code that has not been committed to the repo. This code space was left unused for a period of time and was automatically deleted. Is it possible to restore it? Thank you very much!

**Comment by joshaber** [comment, marked as answer]
It's possible within a certain time window, unfortunately I don't see any restorable codespaces on your user account.

**Comment by LorrinWWW** [comment]
Thank you anyway
